symfony/plugins/orangehrmPimPlugin/Dao/EmployeeEducationDao.php

```diff
@@ -0,0 +1,155 @@
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+namespace OrangeHRM\Pim\Dao;
+
+use Exception;
+use OrangeHRM\Core\Dao\BaseDao;
+use OrangeHRM\Core\Exception\DaoException;
+use OrangeHRM\Entity\EmployeeEducation;
+use OrangeHRM\ORM\Paginator;
+use OrangeHRM\Pim\Dto\EmployeeEducationSearchFilterParams;
+
+class EmployeeEducationDao extends BaseDao
+{
+    /**
+     * @param EmployeeEducation $employeeEducation
+     * @return EmployeeEducation
+     * @throws DaoException
+     */
+    public function saveEmployeeEducation(EmployeeEducation $employeeEducation): EmployeeEducation
+    {
+        try {
+            $this->persist($employeeEducation);
+            return $employeeEducation;
+        } catch (Exception $e) {
+            throw new DaoException($e->getMessage(), $e->getCode(), $e);
+        }
+    }
+
+    /**
+     * @param int $empNumber
+     * @param int $educationId
+     * @return EmployeeEducation|null
+     * @throws DaoException
+     */
+    public function getEmployeeEducationById(int $empNumber, int $educationId): ?EmployeeEducation
+    {
+        try {
+            $employeeEducation = $this->getRepository(EmployeeEducation::class)->findOneBy(
+                [
+                    'employee' => $empNumber,
+                    'education' => $educationId,
+                ]
+            );
+            if ($employeeEducation instanceof EmployeeEducation) {
+                return $employeeEducation;
+            }
+            return null;
+        } catch (Exception $e) {
+            throw new DaoException($e->getMessage(), $e->getCode(), $e);
+        }
+    }
+
+    /**
+     * @param int $empNumber
+     * @param array $toDeleteIds
+     * @return int
+     * @throws DaoException
+     */
+    public function deleteEmployeeEducations(int $empNumber, array $toDeleteIds): int
+    {
+        try {
+            $q = $this->createQueryBuilder(EmployeeEducation::class, 'ee');
+            $q->delete(EmployeeEducation::class, 'ee')
+                ->andWhere('ee.employee = :empNumber')
+                ->andWhere($q->expr()->in('ee.education', ':ids'))
+                ->setParameter('ids', $toDeleteIds)
+                ->setParameter('empNumber', $empNumber);
```

Comment:
can you move this line `->setParameter('empNumber', $empNumber);` after `->andWhere('ee.employee = :empNumber')`